src/utils/local/trigger-on-keys.js

```diff
@@ -16,7 +16,7 @@ import { castArray, compact } from 'lodash'
 function triggerOnKeys(fn, keyCodes) {
   const codes = compact(castArray(keyCodes))
   return function (e) {
-    const key = e.which || e.keyCode
+    const key = e.key || e.code
```

Comment:
The `user.keyboard({Enter})` interaction wasn't passing for this after we switched to using `keydown`. Turns out we were using non-standard & deprecated codes. Switched to what they recommended: https://github.com/testing-library/user-event/issues/842.

This shouldn't be considered a breaking change IMO because `triggerOnKeys` is used internally